torch/quantization/quantize_fx.py

```diff
@@ -91,22 +91,18 @@ def _quantize_fx(model, qconfig_dict, run_fn=None, run_args=None, inplace=False,
 
     if is_dynamic_quant:
         model = prepare_dynamic_fx(model, qconfig_dict, inplace)
-        # TODO: change inplace to True since the model is already copied in
-        # prepare
-        model = convert_dynamic_fx(model, False, debug)
+        model = convert_dynamic_fx(model, inplace=True, debug=debug)
```

Comment:
should this be `inplace=inplace`?

Reply:
this should be True since the inplace option is already applied in the previous step(prepare)